examples/demo-apps/android/LlamaDemo/app/build.gradle.kts

```diff
@@ -57,7 +57,7 @@ dependencies {
   implementation("androidx.constraintlayout:constraintlayout:2.2.0-alpha12")
   implementation("com.facebook.fbjni:fbjni:0.5.1")
   implementation("com.google.code.gson:gson:2.8.6")
-  implementation(files("libs/executorch-llama.aar"))
+  implementation(files("libs/modified-executorch-llama.aar"))
```

Comment:
I can revert this back prior to landing once we have landed the other dependencies.